feat(2022): add source dir message

yoshwata opened the pull request:
## Context

<!-- Why do we need this PR? What was the reason that led you to make this change? -->
Add feature https://github.com/screwdriver-cd/screwdriver/issues/2022

## Objective

<!-- What does this PR fix? What intentional changes will this PR make? -->
Below is example:
<img width="515" alt="Screen_Shot_2020-10-15_at_16_12_56" src="https://user-images.githubusercontent.com/32473622/96089751-3ea0aa00-0f02-11eb-96f2-882edb57909b.png">


## References

<!-- Links or resources that help clarify and support your intentions (e.g., Github issue) -->
https://github.com/screwdriver-cd/screwdriver/issues/2022
https://github.com/screwdriver-cd/screwdriver/issues/2005

https://github.com/screwdriver-cd/notifications-slack/pull/32

## License

<!-- The following line must be included in your pull request -->

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

kumada626 commented:
I think it is better to add sourcePath info to subject because subject includes scmRepo info.
It is better for user that sourcePath info is close to scmRepo info.

yoshwata commented:
@kumada626 
> I think it is better to add sourcePath info to subject

It seems to be reasonable. I've fixed.
Fixed image is below:
<img width="746" alt="Screen_Shot_2020-10-16_at_9_47_33" src="https://user-images.githubusercontent.com/32473622/96201861-a0135800-0f98-11eb-843c-6705f61be9f6.png">
